app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java

```diff
@@ -126,13 +132,16 @@ public void runFunctionalities() {
         boolean guestPlayer = getIntent().getBooleanExtra("guestPlayer", false);
         setGuestPlayerFields(guestPlayer);
 
+        boolean isLocationMocked = getIntent().getBooleanExtra("isLocationMocked", false);
```

Comment:
Same here, mock location attributes don't belong in non-test files.

Reply:
Should I change this and rethink the whole test mechanism?

Reply:
I think so yeah, I will have a look at it this afternoon to see if I can come up with a solution

Reply:
Thank you